scripts/superdesk-authoring/authoring.js

```diff
@@ -61,9 +61,9 @@
 
     var DEFAULT_SCHEMA = Object.freeze({
         slugline: {maxlength: 24},
-        headline: {maxlength: 64, order: 1, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor']},
-        abstract: {maxlength: 160, order: 2, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor']},
-        body_html: {order: 5, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor', 'embed', 'picture', 'table']},
```

Comment:
Yes, you understood it well.

Yes, values change over time and values are changed by values that are set in content_type.json. 

https://github.com/superdesk/superdesk-client-core/blob/e8ab4e3046eed2aa4a3444b005f93b095bf55ef2/scripts/superdesk-authoring/authoring.js#L2314, this type.chema is setting from content_type.json.